v22.1/backup-and-restore-overview.md

```diff
@@ -20,20 +20,20 @@ For an explanation of how a backup works, see [Backup Architecture](backup-archi
 
 {% include cockroachcloud/ccloud/backup-types.md %} 
 
-## Backup and restore types
+## Backup and restore product support
 
-The following table outlines the available backup and restore types in CockroachDB. See each of the pages linked in the table for usage examples:
+This table outlines the level of product support for backup and restore features in CockroachDB. See each of the pages linked in the table for usage examples:
+
+Backup / Restore  | Description  | Product Support | CockroachDB Cloud Backup Type Support
```

Comment:
nit: The fonts in the last two columns look larger than the rest of the table?

Reply:
I know :( it's an unfortunate limitation with styling in tables in the docs. I will play around with trying to get this a little more consistent.

Reply:
I see this too, but ... CSS looks the same for both. Not sure where this comes from, maybe something for Nick. So strange!!